### PR TITLE
[JS]Topic:Documentation for difference between const, var and let #395

### LINKED ISSUE #395: JS: Documentation for difference between const, var and let
I would like to work on explaining this thing . Please assign this to me.Thanks :)

### PR DESCRIPTION
**Pull request by Raghavshalu**
<hr>

## Description 📜
Differences between const, var and let with their syntax and examples..........it includes why const and let is more preferable in that time.
JS: Documentation for difference between const, var and let Fixes #395.

<hr>

## Type of change 📝

- [x] Documentation (Content Creation in the form of codes or tutorials)

<hr>

## Domain of Contribution 📊

- [x] JavaScript

<hr>

## Checklist ✅

- [x] I follow [Contributing Guidelines](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CONTRIBUTING.md) & [Code of conduct](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CODE_OF_CONDUCT.md) of this project.
- [x] I have performed a self-review of my own code or work.
- [x] I have commented on my code, particularly in hard-to-understand areas.
- [x] My changes generates no new warnings.
- [x] I'm a GWOC'21 contributor
 
<hr>

### CODE REVIEW

File: JavaScript/Topics/5.Syntax/Variable_Initialization/README.md [hunk 1]
```diff
@@ -0,0 +1,148 @@
+# Difference between const, var and let
```
Comment (Creatoon) [inline JavaScript/Topics/5.Syntax/Variable_Initialization/README.md:1]:
```suggestion
# Difference between var, let & const
```
Write in correct order of arrival

File: JavaScript/Topics/5.Syntax/Variable_Initialization/README.md [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+There are 3 ways to declare a JavaScript variable:
```
Comment (Creatoon) [inline JavaScript/Topics/5.Syntax/Variable_Initialization/README.md:2]:
```suggestion
There are 3 ways to declare a JavaScript variables:
```

File: JavaScript/Topics/5.Syntax/Variable_Initialization/README.md [hunk 3]
```diff
@@ -0,0 +1,148 @@
...
+- Using `var`
+- Using `let`
+- Using `const`
+## Variables
+**Variables are containers for storing data (values).**
+> Declaring (Creating) JavaScript Variables
+Creating a variable in JavaScript is called "declaring" a variable.
+You declare a JavaScript variable with the `var` keyword:
+
+    var carName;
```
Comment (Creatoon) [inline JavaScript/Topics/5.Syntax/Variable_Initialization/README.md:13]:
```suggestion
var carName;
```
Why space

File: JavaScript/Topics/5.Syntax/Variable_Initialization/README.md [hunk 4]
```diff
@@ -0,0 +1,148 @@
...
+
+    var carName = "Volvo";
+In the example below, we create a variable called carName and assign the value "Volvo" to it.
+
+Then we "output" the value inside an HTML paragraph with id="demo":
+#### Example
+    <p id="demo"></p>
+    <script>
+    var carName = "Volvo";
+    document.getElementById("demo").innerHTML = carName;
```
Comment (Creatoon) [inline JavaScript/Topics/5.Syntax/Variable_Initialization/README.md:29]:
```suggestion
    document.getElementById("demo").textContent = carName;
```

carName is just a string not an html element

**Review by Creatoon — CHANGES_REQUESTED** [review]

**Comment by Raghavshalu** [comment]
@Creatoon Now it is ohk?

File: JavaScript/Topics/5.Syntax/Variable_Initialization/README.md [hunk 5]
```diff
@@ -0,0 +1,148 @@
...
+var carName;
+After the declaration, the variable has no value (technically it has the value of undefined).
+
+To assign a value to the variable, use the equal sign:
+
+    carName = "Volvo";
+You can also assign a value to the variable when you declare it:
+
+    var carName = "Volvo";
+In the example below, we create a variable called carName and assign the value "Volvo" to it.
```
Comment (Creatoon) [inline JavaScript/Topics/5.Syntax/Variable_Initialization/README.md:22]:
```suggestion
In the example above, we create a variable called carName and assign the value "Volvo" to it.
```

I think you are talking about above example

File: JavaScript/Topics/5.Syntax/Variable_Initialization/README.md [hunk 6]
```diff
@@ -0,0 +1,148 @@
...
+
+Then we "output" the value inside an HTML paragraph with id="demo":
+#### Example
+    <p id="demo"></p>
+    <script>
+    var carName = "Volvo";
```
Comment (Creatoon) [inline JavaScript/Topics/5.Syntax/Variable_Initialization/README.md:28]:
```suggestion
      var carName = "Volvo";
```

File: JavaScript/Topics/5.Syntax/Variable_Initialization/README.md [hunk 7]
```diff
@@ -0,0 +1,148 @@
...
+    document.getElementById("demo").textContent = carName;
```
Comment (Creatoon) [inline JavaScript/Topics/5.Syntax/Variable_Initialization/README.md:29]:
```suggestion
      document.getElementById("demo").textContent = carName;
```

Comment (Creatoon) [reply JavaScript/Topics/5.Syntax/Variable_Initialization/README.md:29]: Indent them properly inside script tag

**Review by Creatoon — CHANGES_REQUESTED** [review]

**Comment by Creatoon** [comment]
@Raghavshalu  Just make these changes, i will approve

**Comment by Raghavshalu** [comment]
@Creatoon i have made those changes please review and approve it.

**Review by Creatoon — APPROVED** [review]

**Comment by paramsiddharth** [comment]
@dakshsethi @s4-2 Please review.

**Review by paramsiddharth — APPROVED** [review]

**Comment by s4-2** [comment]
@Raghavshalu  Please write issue number correctly as there should be a gap **Fixes #395**

**Comment by s4-2** [comment]
@Raghavshalu Add your name as a contributor at the end of the file.

**Comment by Raghavshalu** [comment]
@s4-2 Done mam.😊

**Review by s4-2 — APPROVED** [review]
@Raghavshalu  Well done, looks to me.

**Pull request merged**